frontend/src/app/components/incoming-transactions-graph/incoming-transactions-graph.component.ts

```diff
@@ -99,12 +242,48 @@ export class IncomingTransactionsGraphComponent implements OnInit, OnChanges {
       },
       xAxis: {
         type: 'category',
+        axisTick: {
+          alignWithLabel: true,
+          lineStyle: {
+            width: 0,
```

Comment:
Why are you setting the tick line width to 0, are you trying to hide it for some reason?

Reply:
I put it again.